lib/controller.js

```diff
@@ -113,7 +114,8 @@ Tessel.get = function(opts) {
     var seekerOpts = {
       timeout: (opts.timeout || 2) * 1000,
       usb: opts.usb,
-      lan: opts.lan
+      lan: opts.lan,
+      authorized: opts.authorized || true
```

Comment:
@rwaldron thanks for this idea - never used `Boolean()` before.
I am used to have three (3) states to work with: 
- undefined (by `x === "undefined"`)
- true (everything with a value what is neither "undefined" nor false `if (x)`)
- false (if it is false: `if (!x)` what only should match if `x === false` means it will **NOT** match if `x === 0`)

The stupid expression `opts.authorized || true` is really a bad case because the `||` is triggered by `"undefined"` and `false`